.release-please-manifest.json

```diff
@@ -1,3 +1,4 @@
 {
-  ".": "0.1.0"
+  ".": "0.2.0",
+  "charts/prometheus-multipathd-exporter": "1.0.0"
```

Comment:
@mnaser do you know the 1.0.0 version come from? if need same with the app version?